index.html

```diff
@@ -1,231 +1,249 @@
+
+        <article id="project-5" class="project-container">
+          <h2 class="project-subtitle">RPM Governor</h2>
+          <p id="description-5" class="project-paragraph">
+            A printer circuit board, which heart is a microcontroller ATmega328, the circuit board was disgned in Eagle and the Software written in
+            VisualStudio with the plugin VisualMicro. The factory JLCPCB manofacture all the gerber files. its principal aim is read the RPM
+            of the engine also the temperature, the status of the lights, the voltage of the battery and by consequence the health of the
+            Alternator. Whether the RPM surpass 3000RPM the system produce an alarm alerting to the driver, if the driver carry on and the engine
+            transgress 3250 RPM the system open a relay and turn off the engine.
+          </p>
+
+          <div>
+            <ul class="stories-technologies">
+              <li class="tec">C++</li>
+              <li class="tec">Eagle Software</li>
+            </ul>
+          </div>
+
+          <a href="#" class="button">See Project</a>
+        </article>
+      </section>
+
+      <section class="about-section">
+        <article class="about-container">
+          <h2 id="about-title">About me</h2>
+          <p id="about-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <a class="button" href="https://ibritech.atlassian.net/wiki/external/120422658/ZjM2MWM0OGVlOWYyNDAzMGJkM2EzYzA1MzU0YjBiMDg?atlOrigin=eyJpIjoiMjlhZTFkM2U4NjRlNDE1MmI1MWYzNjFmNzJiNWJmNTAiLCJwIjoiYyJ9">
+            Get My Resume
+          </a>
+        </article>
+
+        <div id="divition"></div>
+
+        <article class="knowledge-container">
+          <h2 id="lang">Language</h2>
+          <ul class="knowledge-list">
+            <li class="li">Ruby</li>
+            <li class="li">Html</li>
+            <li class="li">Css</li>
+          </ul>
+
+          <h2 id="frame">Frameworks</h2>
+          <ul class="knowledge-list">
+            <li class="li">Bootstrap</li>
+            <li class="li">Ruby on Rails</li>
+            <li class="li">RSPec</li>
+            <li class="li">CapyBara</li>
+            <li class="li">Selenium</li>
+          </ul>
+
+          <h2 id="skil">Skills</h2>
+          <ul class="knowledge-list">
+            <li class="li">CodeKit</li>
+            <li class="li">GitHub</li>
+            <li class="li">CodePen</li>
+            <li class="li">GitLab</li>
+            <li class="li">Terminal</li>
+          </ul>
+        </article>
+      </section>
     </main>
-    
+
+    <footer class="contact-container">
+      <h3>I'm always interested in hearing about new projects, so if you'd like touch at please get in touch.</h3>
```

Comment:
- [ Optional ] The content of this HTML element is a paragraph. Thus, I believe using `<p>` tag is semantically more appropriate. If you would like to add a section heading you can add something like `Contact Me` heading. what do you think?

Reply:
Ok great thank you i will change it!